source/ref_installation.md

```diff
@@ -0,0 +1,415 @@
+    type: docker
+    uri: gcr.io/my-gcp-project-id-zzzzz/odahu-models-repo
+    username: _json_key
+    webUILink: https://gcr.io/my-gcp-project-id-zzzzz/odahu-models-repo
+- id: models-output
+  spec:
+    description: Object storage for trained artifacts
+    keySecret: '{"auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs", "auth_uri":"https://accounts.google.com/o/oauth2/auth", "client_email":"service-account@my-gcp-project-id-zzzzz.iam.gserviceaccount.com","client_id":"000000000000000000000", "client_x509_cert_url":"https://www.googleapis.com/robot/v1/metadata/x509/service-account%40my-gcp-project-id-zzzzz.iam.gserviceaccount.com", "private_key":"-----BEGIN PRIVATE KEY-----\nprivate-key-here\n-----END PRIVATE KEY-----\n", "private_key_id":"0bacc0b0caa0a0aacabcacbab0a0b00ababacaab", "project_id":"my-gcp-project-id-zzzzz", "token_uri":"https://oauth2.googleapis.com/token", "type":"service_account"}'
+    region: my-gcp-project-id-zzzzz
+    type: gcs
+    uri: gs://odahu-flow-test-store/output
+    webUILink: https://console.cloud.google.com/storage/browser/odahu-flow-test-store/output?project=my-gcp-project-id-zzzzz
+EOF
+```
+
+Install ODAHU services with [Helm chart](https://github.com/odahu/odahu-flow/tree/develop/helms/odahu-flow-core):
+```bash
+$ helm install odahu/odahu-flow-core --name odahu-flow --namespace odahu-flow --values ./odahuflow_values.yaml
+```
+
+### 3.2. Delete ODAHU services
+
+To delete and purge Helm chart run:
+```bash
+$ helm delete --purge odahu-flow
+```
+
+To clean up remaining `CustomResourceDefinitions` execute following command:
+```bash
+$ kubectl get crd | awk '/odahuflow/ {print $1}' | xargs -n1 kubectl delete crd
+```
+
+To purge everything with single command, run
+```bash
+$ helm delete --purge odahu-flow &&\
+  kubectl delete namespace odahu-flow &&\
+  kubectl delete namespace odahu-flow-deployment || true &&\
+  kubectl get crd | awk '/odahuflow/ {print $1}' | xargs -n1 kubectl delete crd &&\
+  kubectl -n istio-system delete job.batch/odahu-flow-feedback-rq-catcher-patcher &&\
+  kubectl -n istio-system delete sa/odahu-flow-feedback-rq-catcher-patcher &&\
+  kubectl -n istio-system delete cm/odahu-flow-feedback-rq-catcher-patch
+```
+
+## 4. Install additional services
+
+### 4.1. MLFlow
+
+```bash
+$ cat << EOF > mlflow_values.yaml
+toolchainVersion: 1.0.0
+logLevel: debug
+ingress:
+  globalDomain: example.com
+  enabled: true
+  tlsEnabled: false
+odahuflowctl:
+  image: "odahu/odahu-flow-cli:1.0.0"
+tracking_server:
+  annotations:
+    sidecar.istio.io/inject: "false"
+  resources:
+    requests:
+      cpu: "100m"
+      memory: "256Mi"
+    limits:
+      cpu: "200m"
+      memory: "512Mi"
+toolchain_integration:
+  enabled: true
+EOF
+```
+
```

Comment:
You forgot the packagers helm chart

Reply:
Done in c35c683823bc4bedc40272cafa686519adcc5856.